src/components/FilterPanel.js

```diff
@@ -1,15 +1,21 @@
+/*eslint no-unused-vars: ["error", { "varsIgnorePattern": "[iI]gnored" }]*/
+
 import React, { useEffect, useState } from "react";
 import { makeStyles } from "@material-ui/core";
 import { useTheme } from "@material-ui/core/styles";
 import useMediaQuery from "@material-ui/core/useMediaQuery";
 import FormControl from "@material-ui/core/FormControl";
-import FormLabel from "@material-ui/core/FormLabel";
+//import FormLabel from "@material-ui/core/FormLabel";
```

Comment:
do you know why this is commented out by chance?

Reply:
It's not being used right now.  I'll can just delete it in the next update.